### DISCUSSION TITLE
1984-08-19-its-not-where-you-start-its-where-you-finish/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#49] [General]
# 1984-08-19-its-not-where-you-start-its-where-you-finish/

Documentary on the stage school where Lena Zavaroni and Bonnie Langford Started

https://fanzoflenazavaroni.github.io/1984-08-19-its-not-where-you-start-its-where-you-finish/

**Comment by StephenMolloy1** [comment]
Someone has commented on YouTube that this is a clip from the programme. 
https://www.youtube.com/watch?v=0UKFZr28Cmc
